services-camel/src/main/java/edu/unc/lib/cdr/fulltext/FulltextRouter.java

```diff
@@ -0,0 +1,46 @@
+package edu.unc.lib.cdr.fulltext;
+
+import static edu.unc.lib.dl.rdf.Fcrepo4Repository.Binary;
+
+import org.apache.camel.BeanInject;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.builder.RouteBuilder;
+
+import edu.unc.lib.cdr.BinaryMetadataProcessor;
+import edu.unc.lib.cdr.FulltextProcessor; 
+
+public class FulltextRouter extends RouteBuilder {
+	@BeanInject(value = "binaryMetadataProcessor")
+	private BinaryMetadataProcessor mdProcessor;
+	
+	@BeanInject(value = "fulltextProcessor")
+	private FulltextProcessor ftProcessor;
+	
+	
+	public void configure() throws Exception {
+		//String mimetypePattern = "^(text/|application/pdf|application/msword|application/vnd\\.|application/rtf|application/powerpoint|application/postscript).*)";
```

Comment:
Yes we should turn this on and make sure it works as expected now that we should be getting mimetypes.